workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css

```diff
@@ -387,7 +391,9 @@ pre[class*='language-'] {
 		table,
 		.table-striped {
 			tbody
-				tr:nth-of-type(odd):not(.table-active):not(.table-disabled):not(.table-divider) {
+				tr:nth-of-type(odd):not(.table-active):not(.table-disabled):not(
```

Comment:
@luisacmn when i run `yarn format` from the workspaces directly it reverts all of these formatting changes. Can you make sure to run that after you run `gw formatSource` from the liferay-learn-workspace directory? Also it might be good for us to ask the product team to fix the formatter so its consistent and does make opposing changes

Reply:
@ryanschuhler , I don't have access to `yarn format` and think Rafa doesn't either. 
Can we talk today about it ?